index.js

```diff
@@ -548,18 +557,21 @@ async function runScan({
     }
   }
 
-  if (failBuild === true && exitCode > 0) {
-    core.setFailed(
-      `Failed minimum severity level. Found vulnerabilities with level ${severityCutoff} or higher`
-    );
-  }
-
   // If there is a non-zero exit status code there are a couple of potential reporting paths
-  if (failBuild === false && exitCode > 0) {
+  if (exitCode > 0) {
     // There was a non-zero exit status but it wasn't because of failing severity, this must be
     // a grype problem
     if (!severityCutoff) {
       core.warning("grype had a non-zero exit status when running");
+    }
+
+    if (failBuild === true) {
+      if (showGrypeOutput) {
+        core.info(cmdOutput);
```

Comment:
I don't believe this is going to give a table-view output but only dumping the SARIF JSON, no?